chunky/src/java/se/llbit/chunky/renderer/scene/Sky.java

```diff
@@ -202,7 +217,6 @@ public void loadSkymap(String fileName) {
    * Set the sky equal to other sky.
    */
   public void set(Sky other) {
-    horizonOffset = other.horizonOffset;
```

Comment:
Did this only not work for simulated sky? We may want to keep it if it works for skymaps (maybe disable it in the GUI if simulated sky is selected).

Reply:
Re-added in 727b301. Changing this requires resetting the sky cache.